verifier.js

```diff
@@ -52,7 +52,15 @@ const PM = exports.PM = new ProcessManager({
 		} catch (e) {}
 
 		return success ? 1 : 0;
-	},
+	}
+}
+
+exports.VerifierManager = VerifierManager;
+
+const PM = exports.PM = new VerifierManager({
+	execFile: __filename,
+	maxProcesses: 1,
```

Comment:
This should probably be a ``const`` at the top for better visibility (or a ``Config`` setting, really)